models/restaurants.go

```diff
@@ -1,27 +1,35 @@
 package models
 
+import "fmt"
+
+// Restaurant data struct
 type Restaurant struct {
 	ID   string `json:"id"`
 	Name string `json:"name"`
 }
 
-func AllRestaurants() ([]*Restaurant, error) {
-	rows, err := db.Query("SELECT * FROM restaurants")
```

Comment:
maybe you changed the create table scripts, what's why errors occurred. So I changed to what exactly the Restaurant struct wants